app/components/Search/SearchContainer.js

```diff
@@ -0,0 +1,142 @@
+import React, { useEffect, useState } from 'react';
+import {
+  Body,
+  Icon,
+  Input,
+  Item,
+  Left,
+  ListItem,
+  Text,
+  View,
+} from 'native-base';
+import { StyleSheet, TouchableOpacity } from 'react-native';
+import theme from '../../theme';
+import Autocomplete from 'native-base-autocomplete';
+import { useDispatch, useSelector } from 'react-redux';
+import * as stockActions from '../Stock/store/actions';
+import * as searchActions from './store/actions';
+import * as watchlistActions from '../Watchlist/store/actions';
+import { Actions as RouteActions } from 'react-native-router-flux';
+import _ from 'lodash';
```

Comment:
use brackets { } and only import what you need from lodash/fp - lodash is quite big!!

Reply:
resolved 4f425b8f74703da8e89917e0ca3e418a69e193e4